src/Murder.Analyzers/DiagnosticIds.cs

```diff
@@ -0,0 +1,13 @@
+namespace Murder.Analyzers;
+
+public static class Diagnostics
+{
+    public static class Resources
+    {
+        public static class ImporterSettingsAttribute
+        {
+            public const string Id = "MRDR0001";
+            public const string Message = "ResourceImporter requires ImporterSettingsAttribute";
```

Comment:
I ended up implementing abstract ResourceImporters :) Will they run okay with this?

Reply:
Just fixed the little conflicts that there were and rebase. They work fine!

Bear in mind I opened the ResourceImporter APIs (after all, if we expect users to implement their own importers, they need to access the APIs)